packages/orchestrator/internal/events/sandbox_events_service.go

```diff
@@ -30,14 +30,14 @@ func (e *MissingEventFieldError) Error() string {
 // as well as persisting using a ClickHouse Batcher
 type SandboxEventsService struct {
 	featureFlags *featureflags.Client
-	pubsub       pubsub.PubSub[event.SandboxEvent, webhooks.SandboxWebhooksMetaData]
```

Comment:
We still need same pub sub impl for internal use during migration period, then we can remove all webhooks metadata related code from there.